html/about.html

```diff
@@ -1 +1,137 @@
+              <li> <a href="#"> Logout </a></li>
+            </ul>
+          </div>
+        </div>
+        <div id="bars" class="nav-btn"><i id="barsIcon" class="fas fa-bars"></i></div>
+      </nav>
+    </header>
+
+    <!--headline  -->
+
+    <section id="headline">
+      <!--backgrounfd-->
+      <div class="head_background">
+        <div class="top_bg">
+          <img src="../img/bg.png" alt="background image">
+        </div>
+        <div class="gamepadIcon bgicon"><i class="fab fa-discord"></i></div>
+        <div class="mouseIcon bgicon"><i class="fas fa-mouse"></i></div>
+        <div class="skullIcon bgicon"><i class="fas fa-skull"></i></div>
+        <div class="skullIcon2 bgicon"><i class="fab fa-playstation"></i></div>
+        <div class="circleIcon "><img src="img/circleIcon.PNG" alt=""></div>
+      </div>
+      <div class="head_content">
+        <div class="headText">
+          <h3>"Hello! shering world"</h3>
+          <h1>Trampoline Dev <br> Global Submit 2022</h1>
+       </div>
+       <div class="head_sup_text"> Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi odit
+          architecto, quis consectetur laborum dolorem voluptatem officiis! </div>
+          <p>Lorem ipsum dolor sit amet consectetur <br> adipisicing elit.</p>
+          <a href="#">sirkorea@gmail.com</a>
+      </div>   
+    </section>
+
+    <section class="logo_info">
+      <h2>Creative Commons Global Summit 2020 Logo</h2>
+      <div class="red_vector"></div>
+      <div class="logo_container"><img src="../img/logo.png" alt="Main Logo"></div>
+    </section>
+
+    <section class="past_submmit">
+      <h2>See the pask TD Global Summits</h2>
+      <div class="red_vector"></div>
+      <p>Take a look at the last two TD Global Summits which took place in Buenos aires and in Warsaw</p>
+      <div class="pastAccess">
+        <div class="2013 submmit"><a href="#"><h2>2013</h2>TD global Summit 2013 in Buenos Aires</a></div>
+        <div class="2011 submmit"><a href="#"><h2>2011</h2>TD Global summit 2011 in Warsar</a></div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="sponsors">
+        <h3>Partner</h3>
+        <div class="red_vector"></div>
+        <div class="logos">
+          <img src="../img/mozilla.png" alt="mozilla" class="sponsor">
+          <img src="../img/Google.png" alt="Google" class="sponsor">
+          <img src="../img/navar.png" alt="Naver" class="sponsor">
+          <img src="../img/Unity-logo.png" alt="3D Unity" class="sponsor unity">
+          <img src="../img/UnrealEngine-logo.png" alt="Unreal Engine" class="sponsor unreal">
+        </div>
+      </div>
+      <div class="copy_right">
+        <div class="footerItem">
+          <img src="../img/footerlogo.png" alt="footer logo">
+          <h3>Trampoline <br/> Dev Honduras</h3>
+        </div>
+        <div class="footerItem">2022 Trampoline Dev Honduras <br/> some Rights Reserved</div>
+      </div>
+    </footer>
```

Comment:
Please ensure that the `About Page footer` has a dark background according to the requirement.